components/layout/Carousel.tsx

```diff
@@ -0,0 +1,48 @@
+import Swiper, { Navigation } from 'swiper'
+import { FC } from 'react'
+import { Token } from 'lib/candy-machine/view/interfaces'
+
+const Carousel: FC<{
+    token: Token[]
```

Comment:
Carousel should be agnostic to NFTs, Tokens or whatever. The Carousel only cares about assets and assets titles